src/basic_recipes/errorbars.jl

```diff
@@ -17,60 +16,44 @@ $(ATTRIBUTES)
         whiskerwidth = 10,
         color = :black,
         linewidth = 1,
-        direction = :y,
+        direction = :vertical,
         visible = theme(scene, :visible)
     )
 end
 
 
-function AbstractPlotting.plot!(plot::Errorbars{T}) where T <: Tuple{<:AbstractVector{<:Real}, <:AbstractVector{<:Real}, <:AbstractVector{<:Real}, <:AbstractVector{<:Real}}
-
-    xs, ys, low, high = plot[1:4]
-    lowhigh = @lift(Point2f0.($low, $high))
-    xys = lift((x, y) -> Point2f0.(x, y), xs, ys)
-    _plot_errorbars!(plot, xys, lowhigh)
-end
-
-function AbstractPlotting.plot!(plot::Errorbars{T}) where T <: Tuple{<:AbstractVector{<:Point2}, <:AbstractVector{<:Real}, <:AbstractVector{<:Real}}
-    xys, low, high = plot[1:3]
-    lowhigh = @lift(Point2f0.($low, $high))
-    _plot_errorbars!(plot, xys, lowhigh)
-end
-
-function AbstractPlotting.plot!(plot::Errorbars{T}) where T <: Tuple{<:AbstractVector{<:Real}, <:AbstractVector{<:Real}, <:AbstractVector{<:Real}}
-    xs, ys, same_lowhigh = plot[1:3]
-    xys = @lift(Point2f0.($xs, $ys))
-    lowhigh = @lift(Point2f0.($same_lowhigh, $same_lowhigh))
-    _plot_errorbars!(plot, xys, lowhigh)
+function AbstractPlotting.convert_arguments(::Type{<:Errorbars}, pos, lower, upper)
+    triplets = broadcast(pos, lower, upper) do p, l, u
+        (p, l, u)
+    end
+    (triplets,)
 end
 
-function AbstractPlotting.plot!(plot::Errorbars{T}) where T <: Tuple{<:AbstractVector{<:Point2}, <:AbstractVector{<:Real}}
-    xys, same_lowhigh = plot[1:2]
-    lowhigh = @lift(Point2f0.($same_lowhigh, $same_lowhigh))
-    _plot_errorbars!(plot, xys, lowhigh)
-end
+function AbstractPlotting.plot!(plot::Errorbars{T}) where T <: Tuple{Any}
 
-f_if(condition, f, arg) = condition ? f(arg) : arg
+    f_if(condition, f, arg) = condition ? f(arg) : arg
 
-function _plot_errorbars!(plot, xys, lowhigh)
+    pos_lows_highs = plot[1]
 
     @extract plot (whiskerwidth, color, linewidth, direction, visible)
 
     is_in_y_direction = lift(direction) do dir
-        if dir == :y
+        if dir == :vertical
             true
-        elseif dir == :x
+        elseif dir == :horizontal
             false
         else
-            error("Invalid direction $dir. Options are :x and :y.")
+            error("Invalid direction $dir. Options are :horizontal and :vertical.")
         end
     end
```

Comment:
I find `:x` and `:y` easier to remember, maybe we could allow both?

Reply:
I actually changed it back just now because I came to the same conclusion after changing xs to pos

Reply:
I don't think it would hurt to just have `dir in (:y, :vertical)` for the people that find that more intuitive